TwoPaneLayout/README.md

```diff
@@ -1,6 +1,6 @@
 # TwoPaneLayout - Surface Duo Compose SDK
 
-**TwoPaneLayout** is a Jetpack Compose component that helps you create UI for dual-screen, foldable, and large-screen devices. TwoPaneLayout provides a two-pane layout for use at the top level of a UI. The component will place two panes side-by-side when the app is spanned on dual-screen, foldable and large-screen devices. These panes can be horizontal or vertical, depending on the orientation of the device and the selected `paneMode`.
+**TwoPaneLayout** is a Jetpack Compose component that helps you create UI for dual-screen, foldable, and large-screen devices. TwoPaneLayout provides a two-pane layout for use at the top level of a UI. The component will place two panes side-by-side when the app is spanned on dual-screen, foldable and large-screen devices, otherwise only one pane will be shown. These panes can be horizontal or vertical, depending on the orientation of the device and the selected `paneMode`.
 
```

Comment:
I would suggest we mention the new `TwoPaneLayoutNav` here to provide a peek at our new feature. And maybe add a direct link, since `TwoPaneLayoutNav` is quite far below. https://github.com/microsoft/surface-duo-compose-sdk/tree/khalp/twopanelayout-updates/TwoPaneLayout#twopanelayoutnav

Reply:
sounds good, that's a good idea!